addons/website/static/src/xml/website.backend.xml

```diff
@@ -103,7 +103,7 @@
     </t>
 
     <t t-name="website.GoToButtons">
-        <a role="button" href="/" class="btn btn-primary" title="Go to Website">
```

Comment:
Don't change that (middle click)

Reply:
Done, sadly it [won't work with ctrl+click](https://superuser.com/questions/854797/why-does-ctrl-click-not-open-some-links-in-a-new-tab) but wheel click will, that's better than nothing